nifi-nar-bundles/nifi-aws-bundle/pom.xml

```diff
@@ -26,10 +26,51 @@
     <packaging>pom</packaging>
 
     <properties>
+        <!-- keep AWS 1.x until NIFI-8287 -->
+        <aws-java-sdk-version>1.12.182</aws-java-sdk-version>
+        <aws-java-sdk-v2-version>2.17.158</aws-java-sdk-v2-version>
```

Comment:
These properties are not necessary since they are declared in the root Maven configuration.

Reply:
I see, thanks